yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -228,6 +241,15 @@ export class Sequencer {
         slot,
       );
 
+      // If I created a "partial" header here that should make our job much easier.
```

Comment:
Coming as part of TxObjects I expect?

Reply:
Not really, but we need the header for the `validateHeader` call so just building it here.